ionic-angular-template/src/auth/sign-in/sign-in.component.html

```diff
@@ -0,0 +1,62 @@
+<ion-header>
+  <ion-toolbar>
+    <ion-title>Login</ion-title>
+  </ion-toolbar>
+</ion-header>
+
+<ion-content class="ion-padding">
+  <form [formGroup]="signInForm" padding>
+    <ion-grid>
+      <ion-row>
```

Comment:
The template in Ionic should follow the following rules: 
`<ion-content>` (optional for scrolling the content)
`<ion-grid>
<ion-row>
<ion-col> `
After that you can have the `<form>` tag or `<divs>` or other content.

Reply:
I got many errors. At the ionic course is the same as mine. And here [](https://www.positronx.io/ionic-angular-responsive-grid-tutorial-with-examples/) as well.

Reply:
`form>
    <ion-grid>
      <ion-row>
        <ion-col size-lg="6" size-md="6" size-sm="6" size="12">
          `....
` </ion-col>
      </ion-row>
    </ion-grid>
  </form>`